gcp_census/model/filesystem_model_provider.py

```diff
@@ -0,0 +1,40 @@
+import json
+import os
+
+from gcp_census.model.table import Table
+from gcp_census.model.view import View
+
+
+class FilesystemModelProvider(object):
+
+    def __init__(self, model_directory):
+        self.model_directory = model_directory
+
+    def list_tables(self):
+        for table in self.__list_files('.json'):
```

Comment:
Why do you assume that files with '.json' extension are tables and with '.sql' are views?

Reply:
That's our convention, as for views we don't need to specify any metadata.